python-scripts/utils/config_utils.py

```diff
@@ -263,8 +263,9 @@ def create_substra_backend_config(org):
             'clientCert': tls_peer_client_dir + '/' + peer['tls']['client']['cert'],
             'clientKey': tls_peer_client_dir + '/' + peer['tls']['client']['key'],
             'grpcOptions': {
-                'grpc-max-send-message-length': 15,
-                'grpc.ssl_target_name_override': peer['host']
+                'grpc.ssl_target_name_override': peer['host'],
+                'grpc.max_send_message_length': -1,
+                'grpc.max_receive_message_length': -1
```

Comment:
Making this the default behavior is kind of bad in my opinion. I can understand that we need to do that temporary in order to take the time it needs to investigate the issue but I disagree on making it the default value. Is this only for docker-compose? is this script used in the kubernetes deployment? (in that case we could use helm values to override this value on some environment and keep the default value to 4Mo which is already big for a ledger call)

Reply:
Yes, I'm waiting for @thibaultrobert  to have more insight on it. See it as a wip/test branch =)